cmd2/cmd2.py

```diff
@@ -95,6 +97,40 @@
     ipython_available = False
 
 
+class CompletionMode(enum.Enum):
```

Comment:
Do these two new classes belong in cmd2.py?  Not sure.  But given the size of cmd2.py, consider a potentially better location.

Reply:
I moved them to utils.py